package.json

```diff
@@ -10,8 +10,14 @@
     "test": "jest"
   },
   "dependencies": {
+    "@react-native-masked-view/masked-view": "^0.2.8",
+    "@react-navigation/native": "^6.1.6",
+    "@react-navigation/stack": "^6.3.16",
```

Comment:
Do we need this for current requirement? If not use the recommended @react-navigation/native-stack

Reply:
I want to use it in the business or restaurant description page and home page navigation as it will look better. so i have used it.